core/gcp/impact.go

```diff
@@ -9,18 +9,12 @@ import (
 )
 
 type CloudIntensity struct {
-	Provider       string
-	ProviderName   string
-	OffsetRatio    int
-	Region         string
-	RegionName     string
-	CountryName    string
-	CountryIsoCode string
-	State          string
-	City           string
-	Impact         float64
-	Source         string
-	GeneralRegion  string
+	Provider      string
+	Region        string
+	Location      string
+	Impact        float64
+	Source        string
+	GeneralRegion string
 }
 
 func loadCloudIntensities(data [][]string) []CloudIntensity {
```

Comment:
This function could be taken out of the example file, and be place in a dedicated module, `import` maybe ?

Reply:
Can you elaborate?  What makes this an "example file"?